legion/legion/containers/docker.py

```diff
@@ -44,103 +42,136 @@ def build_docker_client(args=None):
     return client
 
 
-def build_docker_image(client, base_image, model_id, model_file, labels,
-                       python_package, python_package_version, python_repository,
-                       docker_image_tag, serving):
+def get_docker_container_id_from_cgroup_line(line):
```

Comment:
why do we want to split it to the separate function?

Reply:
I've just split reading and processing data